airflow/cli/commands/standalone_command.py

```diff
@@ -81,6 +81,8 @@ def run(self):
             command=["triggerer"],
             env=env,
         )
+
+        self.web_server_port = int(conf.get('webserver', 'WEB_SERVER_PORT', fallback='8080'))
```

Comment:
Moved it here , because is_ready is called multiple times and to avoid reading the value from the config multiple times.